packages/graph/Makefile.toml

```diff
@@ -23,13 +8,23 @@ args = ["compose", "--env-file", "${REPO_ROOT}/packages/graph/.env", "up", "migr
 cwd = "${REPO_ROOT}/packages/graph/deployment"
 
 [tasks.deployment-up]
+run_task = { name = ["docker-compose-deployment-up", "migrate-db"] }
+
+[tasks.docker-compose-deployment-up]
 private = false
 category = "Deploy"
 description = "Spins up the deployment environment"
 extend = "docker"
 args = ["compose", "--env-file", "../.env", "up", "--wait", "--build"]
 cwd = "${REPO_ROOT}/packages/graph/deployment"
 
+[tasks.migrate-db]
+private = false
+category = "Deploy"
+extend = "run-task"
+description = "Run DB migrations using the Graph binary"
+args = ["run", "--manifest-path", "./hash_graph/Cargo.toml", "--", "migrate"]
```

Comment:
As this is not a private task, do we want to add a `cwd` argument here as well?

Reply:
Oh thank you for mentioning that, that may simplify the way I handle the inconsistency with the manifest path. Will look into this

Reply:
Thank you a lot, this is much better 0c40fda7593f4a3b5ebdf84d33147b9a8776b65b